newrelic-php-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.deamonset.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "newrelic-php-agent.fullname" . }}
+  labels:
+    app: {{ template "newrelic-php-agent.name" . }}
+    chart: {{ template "newrelic-php-agent.chart" . }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ include "newrelic-php-agent.fullname" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "newrelic-php-agent.fullname" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          volumeMounts:
+            - mountPath: /tmp
```

Comment:
`hostPath`(in host) and `mountPath`(in container) are different, is it correct ?

Reply:
@cw-sakamoto It is not a mistake. Because tmp contains things other than newrelic.